src/components/mypage/TabGroup.tsx

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from 'react';
+
+export default function TabGroup({ children }: { children: ReactNode }) {
```

Comment:
children만 prop으로 받으면 이렇게 하는 방법도 있어!
```suggestion
import type { PropsWithChildren } from 'react';

export default function TabGroup({ children }: PropsWithChildren) {
```